src/Model/MyParcelRequest.php

```diff
@@ -167,7 +168,13 @@ public function sendRequest($method = 'POST', $uri = self::REQUEST_TYPE_SHIPMENT
         $request->close();
 
         if ($this->getError()) {
-            throw new ApiException('Error in MyParcel API request: ' . $this->getError() . ' Url: ' . $url . ' Request: ' . $this->body);
+            switch ($this->errorCode->first()) {
+                case 3716:
```

Comment:
Je moet voor 3716 een contante van maken. Iets van ERROR_CODE_SHIPMENT_LIMIT_EXCEEDED of ERROR_CODE_ACCOUNT_NOT_ACTIVE

Reply:
Gedaan.